addons/web/static/src/js/widgets/date_picker.js

```diff
@@ -29,7 +29,7 @@ var DateWidget = Widget.extend({
         this.options = _.extend({
             locale: moment.locale(),
             format : this.type_of_date === 'datetime' ? time.getLangDatetimeFormat() : time.getLangDateFormat(),
-            minDate: moment({ y: 1900 }),
+            minDate: moment({ y: 1000 }),
```

Comment:
Why 1000 and not any lower year?

Reply:
Hi @pedrobaeza, because then you face other issues if you have only 3 digits in the number instead of 4, and fixing these go beyond my capabilities !

Reply:
Uhm, curious that that happens, as it should parse full string.